Dockerfile-alpine

```diff
@@ -62,7 +63,8 @@ RUN echo '@edge http://dl-cdn.alpinelinux.org/alpine/edge/testing' >> /etc/apk/r
     ln -s /usr/bin/php7 /usr/bin/php && \
     ln -s /usr/sbin/php-fpm7 /usr/sbin/php-fpm7.0 && \
     # Disable xdebug by default \
-    sed -i 's/zend_extension\s\?=/;zend_extension =/' $CONF_PHPMODS/xdebug.ini
+    sed -i 's/zend_extension\s\?=/;zend_extension =/' $CONF_PHPMODS/xdebug.ini && \
+    sed -i 's/extension\s\?=/;extension =/' $CONF_PHPMODS/20_redis.ini
```

Comment:
Maybe add a comment just like you did here? https://github.com/bryanlatten/docker-php/pull/89/files#diff-f50ab8c437dc02a64a01f0ff3c3967bfR67